src/Sarif/Readers/RuleDictionaryConverter.cs

```diff
@@ -6,9 +6,13 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Readers
 {
-    public class IRuleDictionaryConverter : JsonConverter
+    /// <summary>
+    /// This converter only exists because the version of Json.NET we're using (6.0.8)
+    /// can't write out a property whose static type is an interface.
+    /// </summary>
+    public class RuleDictionaryConverter : JsonConverter
     {
-        public static readonly IRuleDictionaryConverter Instance = new IRuleDictionaryConverter();
+        public static readonly RuleDictionaryConverter Instance = new RuleDictionaryConverter();
```

Comment:
Renamed because by .NET naming guidelines, something starting with `I` is an interface.